test/e2e/security_test.go

```diff
@@ -169,29 +167,38 @@ func testCert(t *testing.T, data *TestData, expectedCABundle string, restartPod
 			t.Log("CA bundle doesn't match the expected one, retrying")
 			return false, nil
 		}
-		clientConfig := restclient.Config{
-			TLSClientConfig: restclient.TLSClientConfig{
-				Insecure:   false,
-				ServerName: certificate.GetAntreaServerNames(certificate.AntreaServiceName)[0],
-				CAData:     []byte(caBundle),
+		cm := corev1.ConfigMap{
+			TypeMeta: metav1.TypeMeta{
+				Kind:       "ConfigMap",
+				APIVersion: "v1",
 			},
```

Comment:
this should *not* be set manually. The `TypeMeta` is implied by the Go type (`corev1.ConfigMap`).